### DISCUSSION TITLE
I can't run a newly created winui app, it says System.Runtime.InteropServices.COMException from Microsoft.WindowsAppRuntime.Bootstrap.Net.dll

### DISCUSSION THREAD

**Discussion by cdytoby** [microsoft/WindowsAppSDK#1935] [Q&A] [answered]
Project file is here:
https://www.dropbox.com/s/8wramdl642bgith/WinUITest.zip?dl=0

This is a new project without any modification, followed winui 3 tutorial. And I simply can't run or debug it. When click "run without debug", VS does nothing. When click debug button, it says this:

```
引发的异常:“System.Runtime.InteropServices.COMException”(位于 Microsoft.WindowsAppRuntime.Bootstrap.Net.dll 中)
程序“[15256] WinUITest.exe”已退出，返回值为 2148734499 (0x80131623)。
```

translation:

```
Exception: "System.Runtime.InteropServices.COMException", from (Microsoft.WindowsAppRuntime.Bootstrap.Net.dll)
Program WinUITest.exe exited, return value 2148734499 
```

Windows version 21H1, VS 2022 17.0.4

**Comment by cdytoby** [comment, marked as answer]
Follow up:

Another help post I created, https://github.com/microsoft/WindowsAppSDK/discussions/1933, is indeed the cause of this error.

**Reply by DrusTheAxe** [reply]
Solved per https://github.com/microsoft/WindowsAppSDK/discussions/1933#discussioncomment-1882397

**Comment by xiazeyu** [comment]
For me, it was because I didn't installed the [Windows App SDK Runtime](https://docs.microsoft.com/zh-CN/windows/apps/windows-app-sdk/downloads#windows-app-sdk-10).
